inc/certificate_item.class.php

```diff
@@ -313,7 +313,8 @@ public static function showForCertificate(Certificate $certificate) {
                   echo "<td class='center' " . (isset($data['is_deleted']) && $data['is_deleted'] ? "class='tab_bg_2_2'" : "") .
                      ">" . $name . "</td>";
                   if (Session::isMultiEntitiesMode()) {
-                     echo "<td class='center'>" . Dropdown::getDropdownName("glpi_entities", $data['entity']) . "</td>";
+                     $entity = ($item->isEntityAssign() ? Dropdown::getDropdownName("glpi_entities", $data['entity']) : '-');
```

Comment:
maybe a CR before `?` and `:` ?
it was painfull to read this part (horizontal scroll needed) on github